third_party/envoy-gateway/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# This is a copy of
+# https://github.com/envoyproxy/gateway/blob/v1.1.2/tools/docker/envoy-gateway/Dockerfile
+# but:
+# - using calico/base instead of distroless
+# - using TARGETARCH instead of TARGETPLATFORM.
+
+ARG CALICO_BASE
+
+FROM --platform=linux/amd64 busybox AS source
+# Create the data directory for eg
```

Comment:
```suggestion
```

Unfinished comment?

Reply:
Actually no, it means "eg" as an abbreviation for envoy gateway.  But I'll improve it!

Reply:
ahhh makes sense. Yeah that acronym wasn't obvious to me :sweat_smile: